examples/framework/preferences/minimalEditor/schemas/primary.js

```diff
@@ -0,0 +1,37 @@
+ /*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+(function () {
+    "use strict";
+
+   /**
+     * Primary Schema
+     * This schema defines the preference(s) edited by this preferences editor:
+     * their names, types, default values, etc.
+     */
+    fluid.defaults("minEditor.primarySchema", {
```

Comment:
Justin, I expanded the primary schema to add more preferences this way: https://github.com/acheetham/infusion/blob/FLUID-5754%2B/examples/framework/preferences/minimalEditor/schemas/primary.js
Is this incorrect? (it works...)